session_test.go

```diff
@@ -310,7 +310,7 @@ func (s *testSessionSuite) TestAutocommit(c *C) {
 
 	checkTxn(c, se, "create table t (id BIGINT PRIMARY KEY AUTO_INCREMENT NOT NULL)", 0)
 	checkAutocommit(c, se, 2)
-	checkTxn(c, se, "set autocommit=0;", 0)
+	checkTxn(c, se, "set autocommit=0;", 1)
```

Comment:
`set autocommit=0;` shouldn't affect the beginning or committing of a transaction.